City-of-Helsinki/helsinki-paatokset
# PP-273: Add doa link to trustee info card

Add link to trustee declaration of affiliation to the trustee info box.

To test: 

- Checkout branch. Run `make drush-cr; make drush-cim`.
- Add new media of type `declaration of affiliation` or edit an existing one. Make sure that there is a reference to a trustee in the content.
- View the trustee page of the trustee you have referenced in the media item. Info box should have a row containing link to the declaration file.
